src/main/java/org/mule/extension/db/internal/domain/connection/mysql/MySqlConnectionProvider.java

```diff
@@ -38,8 +37,7 @@
 @DisplayName("MySQL Connection")
 @Alias("my-sql")
 @ExternalLib(name = "MySQL JDBC Driver", description = "A JDBC driver which supports connecting to the MySQL Database",
-    nameRegexpMatcher = DRIVER_FILE_NAME_PATTERN, requiredClassName = MYSQL_DRIVER_CLASS, type = JAR,
-    coordinates = MYSQL_GAV)
+    nameRegexpMatcher = DRIVER_FILE_NAME_PATTERN, type = JAR, coordinates = MYSQL_GAV)
```

Comment:
shouldn't the MYSQL_GAV property be updated?

Reply:
By profiling the new version, 8.0.19, the app leaves more than one MuleApplicationClassLoader. Should we leave the current version for the coordinates (5.1.48) or test previous versions to 8.0.19?

Reply:
What do you mean more than one classloader? At most, the application will leak one ApplicationClassLoader. In either case, you should use a profiler to identify the cause of the leak. You should do this tests with both versions.

Reply:
I'm sorry: after multiple iterations in the profiler where I redeploy the app several times and forced the garbage collector at the end, just one instance of the MuleApplicationClassLoader was left - this was consistent in all the iterations.
So I will update the MYSQL_GAV property.

Reply:
there has to be zero. You have a leak

Reply:
I have performed the following tests within a standalone instance with a MuleApp which makes a Select operation against a MySql DB and logs the result:
- First Test -
1. Deployed the MuleApp, the profiler shows just one MuleApplicationClassLoader instance;
2. Modified the Select operation within the MuleApp's XML multiple times, thus forcing multiple redeploys of the MuleApp. This increases MuleApplicationClassLoader instances.
3. Forced the GC resulting in the decrease of the MuleApplicationClassLoader instances, thus leaving just one.
4. Finally, deleted the MuleApp by deleting the anchor-txt file (which deletes the MuleApp's folder) and forced the GC. The result of this is no MuleApplicationClassLoader instance being left whatsoever.

- Second Test -
1. Deployed the MuleApp, the profiler shows just one MuleApplicationClassLoader instance;
2. Deleted the MuleApp by deleting the anchor-txt file (which deletes the MuleApp's folder) and forced the GC. The result of this is no MuleApplicationClassLoader instance being left whatsoever.

Is this consistent with having no memory leaks? Or should I perform other tests?

Reply:
In the first test, without forcing the applicationo's redeploy, make sure to first execute the flow and have the connector actually execute the query.

Reply:
It has been checked that the Selected operation is executed since it's triggered by a scheduler (and the MuleApp's log, <standalone folder>/logs/<MuleApp's name>.log, shows the Select's result).